pycbc/events/stat.py

```diff
@@ -567,6 +569,46 @@ def single(self, trigs):
                            dtype=numpy.float32), ndmin=1, copy=False)
 
 
+class ExpFitSGCoincRateStatistic(ExpFitStatistic):
+
+    """Detection statistic using an exponential falloff noise model.
+    Statistic calculates the log noise coinc rate for each
+    template over single-ifo newsnr values.
+    """
+
+    def __init__(self, files, benchmark_lograte=-14.6):
+        # benchmark_lograte is log of a representative noise trigger rate
+        # This comes from H1L1 (O2) and is 4.5e-7 Hz
+        super(ExpFitSGCoincRateStatistic, self).__init__(files)
+        self.benchmark_lograte = benchmark_lograte
+        self.get_newsnr = ranking.get_newsnr_sgveto
+        # Reassign the rate as the we are now calculating total number in
```

Comment:
rewrite comment - 'as the we' & I would just say that 'rate' data is now number per time, rather than an arbitrarily normalized number, don't say anything about total number or threshold here

Reply:
Comment updated